kuma/redirects/redirects.py

```diff
@@ -705,6 +705,16 @@ def locale_redirect(pattern, to, prepend_locale=True, **kwargs):
         re_flags="i",
         permanent=True,
     ),
+    # All of the remaining "samples/" URL's are redirected to the
+    # the production media domain ("media.prod.mdn.mozit.cloud"),
+    # since the "samples/" legacy files on that domain are the same,
+    # and will always be the same, between stage and production.
+    redirect(
+        r"^samples/(?P<sample_path>.*)$",
+        f"https://media.prod.mdn.mozit.cloud/samples/{{sample_path}}",
```

Comment:
Should we consider using the `ATTACHMENTS_AWS_S3_CUSTOM_DOMAIN` config instead? That should point to the attachments domain instead of actually hard coding the redirect for different environments to a singular domain